### ISSUE TITLE
More medical objects that would make the life easier for those trying to save the humanoid

### ISSUE THREAD

**Issue by Lordjakob** [studio-minus/ppg-bugs-and-requests#2884]
### Feature description

3 Objects , 1 Oxygen tank : a Explosive tank with a face mask on a tube attached that hangs freely like the pads of the defibrillator .it should be able to be attached to the face of a human and when activated will supply the human with oxygen , the only time it wont supply oxygen is when the human lung is perforated , 2 a Electrocauterization machine : it would be a small machine with a small metal "piece" (sorry couldnt find a better word) that would be attached with a wire the same way as the previous object , the metal piece would have the properties of the knife and could stab into a humans body parts when the machine is activated the body parts bleeding , internal bleeding , and if it is the torso and there is a perforated lung it will seal these wounds and fix the human , and the last object is a 3 Splint : this is a simple Object that would attach to limbs that would still heal broken bones . And this is all.

### Feature purpose

These would give ways of healing a human without a life syringe and would have some cool interactions with humanoids like useing the O2 tank for underwater breathing.

**Comment by DindinYT37** [comment]
Duplicate of or Related to:
* #412
* #423
* #541
* #610 
* #787 
* #1085 
* #1220 
* #1709 
* #1916
* #2283
* #2530
* #2678 

These are just for one of the three things you named. Additionally, please don't multi-request, as it makes it hard to accept a request.

**Comment by Lordjakob** [comment]
> Duplicate of or Related to:
> 
>     * [Oxygen/Blood/Circulatory Rework, Volume no2 #412](https://github.com/studio-minus/ppg-bugs-and-requests/issues/412)
> 
>     * [Gases #423](https://github.com/studio-minus/ppg-bugs-and-requests/issues/423)
> 
>     * [Space and Oxygen #541](https://github.com/studio-minus/ppg-bugs-and-requests/issues/541)
> 
>     * [Balloons can store liquids + maybe helium? #610](https://github.com/studio-minus/ppg-bugs-and-requests/issues/610)
> 
>     * [Oxygenator #787](https://github.com/studio-minus/ppg-bugs-and-requests/issues/787)
> 
>     * [oxygen tanks or somethin #1085](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1085)
> 
>     * [Depletable Oxygen Tank/Syringe #1220](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1220)
> 
>     * [Expansion of the respiratory system #1709](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1709)
> 
>     * [Oxygen tank or other gas tanks #1916](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1916)
> 
>     * [Oxygen mask/Breathing machine #2283](https://github.com/studio-minus/ppg-bugs-and-requests/issues/2283)
> 
>     * [Oxygen Tank #2530](https://github.com/studio-minus/ppg-bugs-and-requests/issues/2530)
> 
>     * [Oxygen Tank #2678](https://github.com/studio-minus/ppg-bugs-and-requests/issues/2678)
> 
> 
> These are just for one of the three things you named. Additionally, please don't multi-request, as it makes it hard to accept a request.

where these Objects suggested to be implemented in the same way as my object . No ? then my suggestion is still valid

**Comment by pokka-tea** [comment]
> > Duplicate of or Related to:
> > ```
> > * [Oxygen/Blood/Circulatory Rework, Volume no2 #412](https://github.com/studio-minus/ppg-bugs-and-requests/issues/412)
> > 
> > * [Gases #423](https://github.com/studio-minus/ppg-bugs-and-requests/issues/423)
> > 
> > * [Space and Oxygen #541](https://github.com/studio-minus/ppg-bugs-and-requests/issues/541)
> > 
> > * [Balloons can store liquids + maybe helium? #610](https://github.com/studio-minus/ppg-bugs-and-requests/issues/610)
> > 
> > * [Oxygenator #787](https://github.com/studio-minus/ppg-bugs-and-requests/issues/787)
> > 
> > * [oxygen tanks or somethin #1085](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1085)
> > 
> > * [Depletable Oxygen Tank/Syringe #1220](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1220)
> > 
> > * [Expansion of the respiratory system #1709](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1709)
> > 
> > * [Oxygen tank or other gas tanks #1916](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1916)
> > 
> > * [Oxygen mask/Breathing machine #2283](https://github.com/studio-minus/ppg-bugs-and-requests/issues/2283)
> > 
> > * [Oxygen Tank #2530](https://github.com/studio-minus/ppg-bugs-and-requests/issues/2530)
> > 
> > * [Oxygen Tank #2678](https://github.com/studio-minus/ppg-bugs-and-requests/issues/2678)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > These are just for one of the three things you named. Additionally, please don't multi-request, as it makes it hard to accept a request.
> 
> where these Objects suggested to be implemented in the same way as my object . No ? then my suggestion is still valid

buddy, dont try arguing against him, all he does is wait until someone has a totally legitimate idea (albeit not all) and comments what he finds by searching for it. If he finds anything even remotely similar, he'll post an entire paragraph telling you why your idea is useless and garbage. No idea if he is affiliated with studio minus but god damn, he should be paid!

**Comment by Lordjakob** [comment]
> > > Duplicate of or Related to:
> > > ```
> > > * [Oxygen/Blood/Circulatory Rework, Volume no2 #412](https://github.com/studio-minus/ppg-bugs-and-requests/issues/412)
> > > 
> > > * [Gases #423](https://github.com/studio-minus/ppg-bugs-and-requests/issues/423)
> > > 
> > > * [Space and Oxygen #541](https://github.com/studio-minus/ppg-bugs-and-requests/issues/541)
> > > 
> > > * [Balloons can store liquids + maybe helium? #610](https://github.com/studio-minus/ppg-bugs-and-requests/issues/610)
> > > 
> > > * [Oxygenator #787](https://github.com/studio-minus/ppg-bugs-and-requests/issues/787)
> > > 
> > > * [oxygen tanks or somethin #1085](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1085)
> > > 
> > > * [Depletable Oxygen Tank/Syringe #1220](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1220)
> > > 
> > > * [Expansion of the respiratory system #1709](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1709)
> > > 
> > > * [Oxygen tank or other gas tanks #1916](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1916)
> > > 
> > > * [Oxygen mask/Breathing machine #2283](https://github.com/studio-minus/ppg-bugs-and-requests/issues/2283)
> > > 
> > > * [Oxygen Tank #2530](https://github.com/studio-minus/ppg-bugs-and-requests/issues/2530)
> > > 
> > > * [Oxygen Tank #2678](https://github.com/studio-minus/ppg-bugs-and-requests/issues/2678)
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > These are just for one of the three things you named. Additionally, please don't multi-request, as it makes it hard to accept a request.
> > 
> > 
> > where these Objects suggested to be implemented in the same way as my object . No ? then my suggestion is still valid
> 
> buddy, dont try arguing against him, all he does is wait until someone has a totally legitimate idea (albeit not all) and comments what he finds by searching for it. If he finds anything even remotely similar, he'll post an entire paragraph telling you why your idea is useless and garbage. No idea if he is affiliated with studio minus but god damn, he should be paid!

Hate those guys all i want is to suggest some stuff i think will fit in the game nicely

**Comment by DindinYT37** [comment]
> > Duplicate of or Related to:
> >
> > * ...
> >
> > These are just for one of the three things you named. Additionally, please don't multi-request, as it makes it hard to accept a request.
> 
> where these Objects suggested to be implemented in the same way as my object . No ? then my suggestion is still valid

Yes. They are quite clearly oxygen tanks that provide oxygen to a Human in one way or another. Also, why have you put the quote into a code-box?

**Comment by ras09191** [comment]
dev probably wont implement any of this stuff as it makes the game more complicated

**Comment by Lordjakob** [comment]
?
these items just fix stuff allready in the game
